app/system/controllers/Extensions.php

```diff
@@ -62,9 +62,6 @@ public function index()
 
     public function edit($action, $vendor = null, $extension = null, $context = null)
     {
-        if (!$this->getUser()->hasPermission('Site.Settings'))
```

Comment:
still needed to only admin with Site.Settings and setting permission access

Reply:
Sorry, not sure what you mean. This one didnt seem to be needed when I tested it as the requiredPermissions already made sure this was enforced

Reply:
Well depends on what the expected behaviour should be, the current requiredPermissions value works for when the user has either Admin.Extensions or Site.Settings permission, adding this check ensures the user has the Site.Settings permission.

Reply:
Got you now, its back.